src/mintpy/utils/utils0.py

```diff
@@ -330,11 +329,9 @@ def latlon2utm(lat, lon):
                 lon      - scalar or 1/2D np.ndarray, WGS 84 coordiantes in x direction
     Returns:    easting  - scalar or 1/2D np.ndarray, UTM    coordiantes in x direction
```

Comment:
Should we cut out the `easting/northing` arggs from the docstring?

Reply:
`easting/northing` are returned parameters here, I would prefer to keep them to help understand the function usage. 

If you meant the docstring style of input/output here is not standardized as the `compass` style, I agree. I would love to change the style in another PR for all functions.